docker-compose.yml

```diff
@@ -39,3 +39,17 @@ services:
   #     - MINIO_ROOT_USER=root
   #     - MINIO_ROOT_PASSWORD=password
   #   command: server --console-address :9001 /data
+
+  mode-provider:
+    image: docker.io/bitnami/etcd:3
+    profiles:
+      - multi-tenant
+    environment:
+      - ALLOW_NONE_AUTHENTICATION=yes
+      - ETCD_NAME=mode-provider-etcd
+      - ETCD_LISTEN_PEER_URLS=http://0.0.0.0:2380
+      - ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379
+      - ETCD_ADVERTISE_CLIENT_URLS=http://mode-provider-etcd:2379
```

Comment:
is `mode-provider-etcd` resolvable? shouldn't use `localhost` instead?

Reply:
it will be inside the network, right?